### DISCUSSION TITLE
Can't disable Ctrl+Q shortcut for quitting

### DISCUSSION THREAD

**Discussion by Andrew15-5** [zellij-org/zellij#3026] [Q&A] [answered]
I used Ctrl+Q in Neovim to enter block selection mode and then suddenly the window is gone. Apparently, I can quit zellij by simply pressing this shortcut. And the only place where I found the `Quit` command is here:

```
    shared_except "locked" {
        bind "Ctrl g" { SwitchToMode "Locked"; }
        bind "Ctrl q" { Quit; }
```

I was able to make `bind "Ctrl w" { Quit; }` and it does work in a new terminal window with zellij, but disabling/commenting the `bind "Ctrl q" { Quit; }` didn't change anything. I reckon it's hardcoded somewhere, so this binding is useless. Although I can use Ctrl + V instead, but I'm not used to it and it's a workaround rather than fix.

**Comment by cristiand391** [comment, marked as answer]
> I reckon it's hardcoded somewhere, so this binding is useless

Yes, zellij loads the default keybindings first and then your defined ones override them, that's why `bind "Ctrl q" { Quit; }` works.


You can either unbind `Ctrl + g` or disable all defaults (`clear-defaults=true`), docs:
https://zellij.dev/documentation/keybindings-binding#overriding-keys

**Reply by Andrew15-5** [reply]
> zellij loads the default keybindings first

> You can either unbind [...] or disable all defaults

Oh, so **I** have to be explicit? I think it's very unnatural since it loads defaults **anyway**, but in my config the "defaults" are removed, so I expect it to not load "the defaults". So the config is rather "documentation" because I can't **remove** some default shortcuts by **removing** them from the config.

In this case, it's better to also "document" the `unbind` and `clear-default=true` features too. There are comments about "you can add new keybinding":

```
        // uncomment this and adjust key if using copy_on_select=false
        // bind "Alt c" { Copy; }
```

But there is no comment about "you can disable the default keybinding". To be fair, there is a comment about disabling all keybinding for an entire group:

```
// If you'd like to override the default keybindings completely, be sure to change "keybinds" to "keybinds clear-defaults=true"
```

But still nothing about `unbind` and that "removing written keybindings won't affect anything".

**Reply by cristiand391** [reply]
yeah, maybe we could add the link to the `overriding keys` docs  in the default config. I would ask either in discord or a new issue.

or if you just want to open a PR, this seems to be the file:
https://github.com/zellij-org/zellij/blob/main/zellij-utils/assets/config/default.kdl
